projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-full-immediately.php

```diff
@@ -1026,9 +1031,9 @@ function test_full_sync_can_sync_individual_comments() {
 	}
 
 	function test_full_sync_can_sync_individual_users() {
```

Comment:
Should we add a test here to confirm that a less-than-contributor is not included?

Reply:
test_full_sync_sends_all_users creates a subscriber and verifies via the count that it doesn't get included. 

I'm of the opinion that new tests have diminishing returns due to Full Sync being replaced with checksums.